reana_commons/serial.py

```diff
@@ -91,7 +93,12 @@ def _expand_parameters(specification, parameters):
             for command_num, command in enumerate(step['commands']):
                 current_step['commands'][command_num] = \
                     Template(command).substitute(parameters)
-        return expanded_specification
+        # if call is done from client, original==True and original
```

Comment:
Let's remove this comment and add to the docstring the documention, there we can add the `original` parameter and the function `return`.